### ISSUE TITLE
Unable to transfer files to librewolf

### ISSUE THREAD

**Issue by Huiaei** [fm-sys/snapdrop-android#220]
### Checklist

- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have updated "Snapdrop for Android" to the latest version. The bug is reproducible on this latest version.
- [X] I have checked, that this is NOT a general Snapdrop bug. It is only related to the Android app, and not reproducible with the website as well.

### App version

v1.11.0

### Android version

12

### Describe the bug

Unable to transfer, please read "Steps" for a detailed description

### Steps to reproduce the bug

1. Select multiple image files from google albums
2. Use "Share" to the app
3. use librewolf to open snapdrop.net
4. transfer (when clicking on the device, the background animation stalls and librewolf doesn't respond)

I have tested the following items
* App transfer to 360 Extreme Browser (a chromium kernel based browser), the transfer works fine.
* librewolf transfer to 360 Extreme Browser, the transfer is normal.
* App transfer to librewolf privacy mode, no response. (To exclude possible effects of librewolf browser plugins, privacy mode only has ublock origin and bitwarden browser plugins)

Version：
* snapdrop for Android v1.11.0
* librewolf portable 102.0.1-1
* 360 Extreme Browser 21.0.1120.0 (chromium 95.0.4638.69)

Translated by DeepL

**Comment by fm-sys** [comment]
Can you please check whether you can send files between Librewolf and  360 Extreme Browser **in both directions**?

From reading the feature description of Librewolf (might be related):
> _Protect the private IP address of the user when WebRTC is used._

https://librewolf.net/docs/features/

**Comment by Huiaei** [comment]
> Can you please check whether you can send files between Librewolf and 360 Extreme Browser **in both directions**?
> 
> From reading the feature description of Librewolf (might be related):
> 
> > _Protect the private IP address of the user when WebRTC is used._
> 
> https://librewolf.net/docs/features/

I'm sorry to reply to you now, as I don't use github much.

As per your request, I tested the bi-directional transfer between Librewolf and 360 Extreme Browser. The test results are as follows.

1. transfer from 360 Extreme Browser to librewolf, the transfer is normal
2. transfer from librewolf to 360 Extreme Browser, there is a problem
3. Transferring from librewolf (privacy mode) to 360 Extreme Browser, the transfer is normal

For the second test, I thought it might be a problem with my proxy, so I conducted a third test, using privacy mode.

Translated by DeepL
